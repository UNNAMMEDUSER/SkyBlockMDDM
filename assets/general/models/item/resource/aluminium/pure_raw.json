{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/aluminium/pure_raw"
	}
}