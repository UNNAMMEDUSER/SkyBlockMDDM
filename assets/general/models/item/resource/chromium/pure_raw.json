{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/chromium/pure_raw"
	}
}