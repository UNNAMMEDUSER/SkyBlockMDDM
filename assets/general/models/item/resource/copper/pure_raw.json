{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/copper/pure_raw"
	}
}