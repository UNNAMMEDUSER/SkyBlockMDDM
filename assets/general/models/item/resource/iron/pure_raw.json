{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/iron/pure_raw"
	}
}