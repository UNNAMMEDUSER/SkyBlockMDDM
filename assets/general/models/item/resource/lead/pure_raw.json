{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/lead/pure_raw"
	}
}