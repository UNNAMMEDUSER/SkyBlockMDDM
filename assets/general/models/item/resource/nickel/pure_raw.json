{
	"credit": "Miwo4ka",
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/nickel/pure_raw"
	}
}