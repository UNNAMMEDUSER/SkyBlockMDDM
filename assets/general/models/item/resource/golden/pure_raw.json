{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/golden/pure_raw"
	}
}